job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/QueueFailure.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.scheduled.executor;
+
+import java.util.Date;
+import java.util.Objects;
+
+public class QueueFailure
```

Comment:
Add a comment to say where this came from - we may want to keep in sync until we move to something more like what we do in storage - i.e. the generated model.